corehq/apps/domain/templates/domain/admin/flags_and_privileges.html

```diff
@@ -89,18 +89,41 @@ <h1>Feature Flags</h1>
         <div class="col-sm-10">
             <table class="table table-striped">
                 <thead>
+                    <th>{% trans "Tag" %}</th>
                     <th>{% trans "Feature" %}</th>
                     <th class="checkmark-column">{% trans "Enabled for domain?" %}</th>
                     <th class="checkmark-column">{% trans "Enabled for me?" %}</th>
                     <th>{% trans "Edit" %}</th>
                 </thead>
                 <tbody>
-                    {% for feature, enabled_for_domain, enabled_for_user in flags %}
+                    {% for toggle, enabled_for_domain, enabled_for_user in flags %}
                     <tr class="{% if enabled_for_domain %}success{% elif enabled_for_user %}info{% endif %}">
-                        <td>{{ feature.label }}</td>
+                        <td><span class="label label-{{ toggle.tag.css_class }}">{{ toggle.tag.name }}</span></td>
+                        <td>
+
+                            {% if toggle.description %}
+                            <a class="collapsed" data-toggle="collapse" aria-expanded="false"
+                               href="#toggleDescription-{{ toggle.slug }}">
+                                <i class="fa fa-angle-double-down"></i>
+                            </a>
+                            {% endif %}
```

Comment:
Note that with this version the user has to click that tiny icon - I may try to make the whole row clickable at some point